test/src/unit-capi-config.cc

```diff
@@ -229,6 +229,7 @@ void check_save_to_file() {
   ss << "rest.curl.buffer_size 524288\n";
   ss << "rest.curl.verbose false\n";
   ss << "rest.http_compressor any\n";
+  ss << "rest.load_enumerations_on_array_open true\n";
```

Comment:
This and related configs should not be included .

Reply:
Yeah this is weird... The instructions from github included commits that shouldn't be there... Fixed now.